release-notes/release-notes54.md

```diff
@@ -187,6 +188,34 @@ To develop and maintain a new spellchecking plugin based on the free {{site.prod
 
 The `table_responsive_width` option has been deprecated with the release of {{site.productname}} 5.4. This option has been replaced by [`table_sizing_mode`]({{site.baseurl}}/plugins/table/#table_sizing_mode).
 
+## Known issues
+
+This section describes issues that users of {{site.productname}} 5.4 may encounter, as well as possible workarounds for these issues.
+
+**Outline**
+
+* [Top and bottom editor anchors appear in links autocomplete list when disabled](#topandbottomeditoranchorsappearinlinksautocompletelistwhendisabled)
+
+### Top and bottom editor anchors appear in links autocomplete list when disabled
+
+#### Issue
+
+This issue affects the following options:
+
+- `anchor_top`
+- `anchor_bottom`
+
+When this options are set to `false`, the relevant anchors should not appear as autocomplete options in the link dialog. These anchors will appear in the autocomplete list when the relevant options are set to `false` in {{site.productname}} 5.4.0 and {{site.productname}} 5.4.1.
```

Comment:
When "these"?

Reply:
fixed in: https://github.com/tinymce/tinymce-docs/pull/1613/commits/1e6c1d0833ea57c8f926e5903187f3b26e731827

Reply:
Not fixed